src/dwn-server.ts

```diff
@@ -22,6 +23,11 @@ export class DwnServer {
     this.dwn = options.dwn;
   }
 
+  async start() : Promise<void> {
```

Comment:
why not call this from main?

Reply:
so we have a method on the class, 
so consumer of the package can do something like 

```
import { DwnServer } from 'dwn-server';

const server = new DwnServer();
server.start();
```

let me know if you have a different vision

Reply:
right, I think we should make `main.ts` be that exact code.

Reply:
oh you mean `start` server instead of `listen`? yeah that makes sense, for some reason I tried to minimize changes, but that makes total sense